moto/sagemaker/models.py

```diff
@@ -74,6 +74,49 @@ def response_object(self):
         return self.gen_response_object()
 
 
+class FakePipeline(BaseObject):
+    def __init__(
+        self,
+        pipeline_name,
+        pipeline_display_name,
+        pipeline_definition,
+        pipeline_description,
+        role_arn,
+        tags,
+        account_id,
+        region_name,
+        parallelism_configuration,
+        pipeline_definition_s3_location,
+    ):
+        self.pipeline_name = pipeline_name
+        self.pipeline_arn = arn_formatter(
+            "pipeline", pipeline_name, account_id, region_name
+        )
+        self.pipeline_display_name = pipeline_display_name
+        self.pipeline_definition = pipeline_definition
+        self.pipeline_description = pipeline_description
+        self.role_arn = role_arn
+        self.tags = tags or []
+        self.parallelism_configuration = parallelism_configuration
+        self.pipeline_definition_s3_location = pipeline_definition_s3_location
+
+        now_string = datetime.now().strftime("%Y-%m-%d %H:%M:%S")
+        self.creation_time = now_string
+        self.last_modified_time = now_string
+        self.last_execution_time = now_string
+
+    @property
+    def response_object(self):
```

Comment:
The `response_object` and `response_create` arent' used, as far as I can tell, so let's remove them

Reply:
Thanks for the review and the comments! All change requests implemented.